ecrtokenrefresher/go.mod

```diff
@@ -0,0 +1,51 @@
+module ecrtokenrefresher
+
+go 1.18
```

Comment:
Can you use go 1.17 instead? I believe this repo is mainly built in go 1.17. If we can keep this one consistent as well, I may be able to combine it into the [existing project](https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/aws/eks-anywhere-packages/Makefile) that has been built out of the build tooling.

Reply:
Changed in latest commit